internal/provider/resource_rbac_permission_user.go

```diff
@@ -0,0 +1,120 @@
+package provider
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	api "goauthentik.io/api/v3"
+)
+
+func resourceRBACUserObjectPermission() *schema.Resource {
+	return &schema.Resource{
+		Description:   "RBAC --- ",
+		CreateContext: resourceRBACUserObjectPermissionCreate,
+		ReadContext:   resourceRBACUserObjectPermissionRead,
+		UpdateContext: resourceRBACUserObjectPermissionUpdate,
+		DeleteContext: resourceRBACUserObjectPermissionDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Schema: map[string]*schema.Schema{
+			"user": {
+				Type:     schema.TypeInt,
+				Required: true,
+				ForceNew: true,
+			},
+			"model": {
+				Type:             schema.TypeString,
+				Required:         true,
+				ForceNew:         true,
+				Description:      EnumToDescription(api.AllowedModelEnumEnumValues),
+				ValidateDiagFunc: StringInEnum(api.AllowedModelEnumEnumValues),
+			},
+			"permission": {
+				Type:     schema.TypeString,
```

Comment:
Would it be possible to pass multiple permissions as a list of string? Otherwise it may break the option to attach multiple permissions to a role, as defining multiple role permissions in terraform may override the current ones.